pkg/algorithm/cell_allocation.go

```diff
@@ -89,21 +90,59 @@ func getLowestFreeCellLevel(freeList ChainCellList, l CellLevel) CellLevel {
 		"even split to the highest level %v", l-1))
 }
 
+// when buddyAlloc allocates bad cells,
```

Comment:
>when buddyAlloc allocates bad cells, [](start = 3, length = 36)

after buddyAlloc failed to allocate cells, ...

Reply:
updated